docs/src/user-guide/guides-using-object-storage/clp-usage.md

```diff
@@ -11,22 +11,21 @@ angle brackets (`<>`) with the appropriate values:
 ```bash
 sbin/compress.sh \
   --timestamp-key <timestamp-key> \
-  <prefix>
+  https://<bucket-name>.s3.<region-code>.amazonaws.com/<prefix>
 ```
 
-* `<prefix>` is the prefix of all logs you wish to compress and must be relative to
-  [logs-input.s3_config.key_prefix][logs-input-s3-config].
-  * E.g., if you want to compress the S3 object `/a/b/c.jsonl`, and
-    `logs-input.s3_config.key_prefix` is `/a/`, then you would replace `<prefix>` in the command
-    above with `b/c.jsonl`.
+* `<timestamp-key>` is the field path of the kv-pair that contains the timestamp in each log event.
```

Comment:
comparing with the previous version, maybe we can remove this line?
```suggestion
```

Reply:
should we update the doc to mention we accept either a path-style URL or virtual-hosted–style URL and refer to this [link](https://docs.aws.amazon.com/AmazonS3/latest/userguide/VirtualHosting.html)? But then we also need to explain that instead of using a "key-name", we only need a <prefix>.

Any opinion @Eden-D-Zhang @kirkrodrigues ?